src/components/VmsList/style.css

```diff
@@ -1,15 +1,7 @@
-.movable-left { /* general transition properties */
+.vm-card-list-container { /* general transition properties */
     transition: all 0.6s;
     transform-origin: 0 50%;
     transition-timing-function: ease-out;
-
-    overflow: auto;
-/*    height: calc(100vh - 49px);*/
-}
-
-.moved-left { /* altered state */
```

Comment:
update the PR / commit to mention deleting some unused classes (and thanks for cleaning up!)